### PR TITLE
move to figshare links

### PR DESCRIPTION
**Pull request by wrightaprilm**
Fix issue #306. Pretty simple, but it would be great to have a second set of eyes on.

### CODE REVIEW

File: setup.md [hunk 1]
```diff
@@ -7,18 +7,18 @@ title: Setup
...
+> - [surveys.csv](https://ndownloader.figshare.com/files/10717177)
+> - [species.csv](https://ndownloader.figshare.com/files/3299483)
 > - [speciesSubset.csv]({{ page.root }}/data/speciesSubset.csv)
 > - [surveys2001.csv]({{ page.root }}/data/survey2001.csv)
 > - [surveys2002.csv]({{ page.root }}/data/survey2002.csv)
-> - [plots.csv]({{ page.root }}/data/plots.csv)
+> - [plots.csv](https://ndownloader.figshare.com/files/3299474)
 > - [bouldercreek_09_2013.txt]({{ page.root }}/data/bouldercreek_09_2013.txt)
-> - [SQL Database]({{ page.root }}/data/portal_mammals.sqlite)
+> - [SQL Database](https://ndownloader.figshare.com/files/11188550)
```
Comment (maxim-belkin) [inline setup.md]: @wrightaprilm, do we want to keep copies of the data files in this repo (in the `data` folder)?

**Comment by maxim-belkin** [comment]
Adding bullet points from #306 here

- [ ] Run through the lessons quickly and make sure they work with the Figshare data before merging
- [ ] Move surveys2001 and 2002 to sample output, or just delete them
- [ ] Add the other two datafiles to a figshare project so we have an external. Alternatively, I think it would be OK to delete speciesSubset, and have them take the first couple rows of the species.csv file as their dataset for the join activity. It's really just provided so there is a small enough dataset to see the different joins clearly and non-overwhelmingly.

File: setup.md [hunk 2]
```diff
@@ -7,18 +7,18 @@ title: Setup
 > Data for this lesson is from the Portal Project Teaching Database -
 > [available on FigShare](https://figshare.com/articles/Portal_Project_Teaching_Database/1314459).
 >
-> We will use the eight files listed below for the data in this lesson.  Download these files to your computer either by clicking [this link ](https://minhaskamal.github.io/DownGit/#/home?url=https://github.com/datacarpentry/python-ecology-lesson/tree/gh-pages/data), which will give you everything in a single compressed file.  You'll need to unzip this file after downloading it.
+> We will use the eight files listed below for the data in this lesson.  Download these files to your computer either by clicking [this link ](https://minhaskamal.github.io/DownGit/#/home?url=https:%2F%2Fgithub.com%2Fweecology%2Fportal-teachingdb), which will give you everything in a single compressed file.  You'll need to unzip this file after downloading it.
```
Comment (maxim-belkin) [inline setup.md]: no need to change `/` to `%2F`

File: _episodes/09-working-with-sql.md [hunk 3]
```diff
@@ -66,7 +66,7 @@ con = sqlite3.connect("data/portal_mammals.sqlite")
 cur = con.cursor()
 
 # Return all results of query
-SELECT plot_id FROM plots WHERE plot_type="Control"')
+cur.execute('SELECT plot_id FROM plots WHERE plot_type="Control"')
```
Comment (maxim-belkin) [inline _episodes/09-working-with-sql.md]: It might be better to do it in a separate PR

Comment (wrightaprilm) [reply _episodes/09-working-with-sql.md]: Just rolled it back.

**Comment by wrightaprilm** [comment]
OK, I ran through everything. It all works with the ndownloader links, so I think we ought to merge this. Regarding points 2 and 3, I think we should just delete the surveys files. We can also delete plots.csv - it's no longer used. I agree with removing speciesSubset and just having them make it.

**Comment by maxim-belkin** [comment]
OK. Let's merge this PR and create a PR that deletes the files you mention.

**Comment by wrightaprilm** [comment]
Sounds like a plan.
